### ISSUE TITLE
Unable to view CMD in the tag

### ISSUE THREAD

**Issue by lueliu** [mkuchin/docker-registry-web#20]
Hi  THX your tool.

When I click on a tag,display,"An error has occurred"

my configuration error ? or something error.

How to display multiple levels of operation in a tag?

test-registry:latest has 26 layers

**docker-registry-web logs**
```
2015-12-30 07:59:06,974 [http-bio-8080-exec-4] ERROR errors.GrailsExceptionResolver  - GroovyRuntimeException occurred when processing request: [GET] /tag/centos/6.6
Ambiguous method overloading for method java.math.BigInteger#plus.
Cannot resolve which method to invoke for [null] due to overlapping prototypes between:
	[class java.lang.Character]
	[class java.lang.String]
	[class java.lang.Number]. Stacktrace follows:
groovy.lang.GroovyRuntimeException: Ambiguous method overloading for method java.math.BigInteger#plus.
Cannot resolve which method to invoke for [null] due to overlapping prototypes between:
	[class java.lang.Character]
	[class java.lang.String]
	[class java.lang.Number]
	at docker.registry.web.RepositoryController.tag(RepositoryController.groovy:65)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
2015-12-30 07:59:19,530 [http-bio-8080-exec-3] ERROR errors.GrailsExceptionResolver  - NullPointerException occurred when processing request: [GET] /tag/test-registry/latest
Stacktrace follows:
java.lang.NullPointerException


```

**Comment by hknochi** [comment]
push

i've the same problem

**Comment by mkuchin** [comment]
It seems it was fixed in #22, could you check if it working for you?
